scripts/test-eunomia.sh

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env bash
+set -eu
+
+deploy_dev_image() {
+	echo "Deploying Dev Image Tag"
+	helm template deploy/helm/eunomia-operator/ | kubectl apply -f -
+	sleep 20
+	kubectl config set-context --current --namespace=eunomia-operator
```

Comment:
1. shouldn't we use a test namespace, as in e2e-test.sh?
2. the test doesn't seem to clean up the namespace before starting, so if there is already operator running in it, the test could misleadingly succeed even if kubectl apply below fails, no?
3. also, doesn't `kubectl config set-context` change the namespace permanently for the user of the script, even after the script ends? I don't think we should do something like that, why not modify the namespace only temporarily with helm, as in e2e-test.sh?